packages/ckeditor5-code-block/src/codeblockcommand.js

```diff
@@ -166,3 +185,26 @@ function canBeCodeBlock( schema, element ) {
 
 	return schema.checkChild( element.parent, 'codeBlock' );
 }
+
+// Picks the language for the new code block. If option usePreviousLanguageChoice
+// is true and some code block was already created (lastLanguage is not null) then previously
+// used language will be returned. Else, it will see if any language is passed as an option
+// and return that. If not, it will return default language.
+//
+// @param {Object} options
+// @param {Boolean} [options.usePreviousLanguageChoice]
+// @param {String} [options.language]
+// @param {String|null} lastLanguage
+// @param {String} defaultLanguage
+// @return {String}
+function getLanguage( options, lastLanguage, defaultLanguage ) {
+	if ( options.usePreviousLanguageChoice && lastLanguage ) {
+		return lastLanguage;
+	}
+
+	if ( options.language ) {
+		return options.language;
+	}
```

Comment:
```suggestion
	if ( options.language ) {
		return options.language;
	}

	if ( options.usePreviousLanguageChoice && lastLanguage ) {
		return lastLanguage;
	}
```

I'd swap the order of these conditions. If the `options.language` is specified, it should be used. Please, do not update the function docs.

Reply:
One more thing, make sure that the change is covered by a test (in the `CodeBlockCommand` file).